src/lib/annotations/Annotator.js

```diff
@@ -236,10 +237,11 @@ class Annotator extends EventEmitter {
      * Sets the zoom scale.
      *
      * @param {number} scale - current zoom scale
-     * @return {void}
+     * @return {Annotator} Annotator instance returned for chaining
      */
     setScale(scale) {
         this.annotatedElement.setAttribute('data-scale', scale);
+        return this;
```

Comment:
I understand the benefits of chaining, but since we already have access to the annotator instance, what is the benefit here over just calling `instance.whatever()` after calling `instance.setScale()`?

My thought process is that it goes against any coding paradigms we currently follow, in Preview